tests/cpp_unit_tests/test_math_solver.cpp

```diff
@@ -8,6 +8,8 @@
 #include "power_grid_model/math_solver/newton_raphson_pf_solver.hpp"
 #include "power_grid_model/three_phase_tensor.hpp"
 
+using namespace std::placeholders;
```

Comment:
Why do you need this?

Maybe put the using namespace inside the `power_grid_model` namespace?

Reply:
I'm using the `_1` and `_2` placeholder in the unit test to sum the `s` attributes of all (one or two) `load_gens`.
```c++
const ComplexValue<true> load_gen_s = std::accumulate(
    output.load_gen.begin(),
    output.load_gen.end(),
    ComplexValue<true>{},
    bind(std::plus<ComplexValue<true>>(), _1, bind(&ApplianceMathOutput<true>::s, _2)));
```
And since it is inside a unit test I didn't really worry about namespaces. But I moved it inside the `power_grid_model ` namespace now.